decorator.rb

```diff
@@ -0,0 +1,11 @@
+require './Nameable'
```

Comment:
- [ ] Please change `require './Nameable'` to `require './nameable'` to fix the code run error since `require './Nameable'` does not load the `nameable.rb` file. :pray: